postmaster/forms.py

```diff
@@ -16,6 +16,7 @@ class LoginForm(Form):
     """
     username = StringField(label='Username', validators=[DataRequired()])
     password = PasswordField(label='Password', validators=[DataRequired(), validate_wtforms_password])
+    two_factor = IntegerField(label='2 Factor', validators=(validators.Optional(),))
```

Comment:
I'm also thinking that instead of this, we should pop up a modal if two factor authentication is enabled. This is fine for now though.